data-raw/dk_activity_units.R

```diff
@@ -0,0 +1,1624 @@
+                   "during lockdown after the opening of 0-4th grade (phase 1)"),
+    home = 0.0,
+    # LAEC work = (rep(c(0, (1069000) / work_force, 0), c(3, 10, 3))),  #
+    work = (rep(c((1069000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.05 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_phase_2",
+    label = "Private employees not directly mentioned and working during lockdown after March 1st (phase 2)",
+    home = 0.0,
+    # LAEC work = (rep(c(0, (1262000) / work_force, 0), c(3, 10, 3))),  #
+    work = (rep(c((1262000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.05 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_phase_4",
+    label = "Private employees not directly mentioned and working during lockdown after May 1st",
+    home = 0.0,
+    # LAEC work = (rep(c(0, (1332000) / work_force, 0), c(3, 10, 3))),  #
+    work = (rep(c((1332000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.06 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_phase_5",
+    label = "Private employees not directly mentioned and working during lockdown after May 21st",
+    home = 0.0,
+    # LAEC work = (rep(c(0, (1489000) / work_force, 0), c(3, 10, 3))),  #
+    work = (rep(c((1489000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.06 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_phase_6",
+    label = "Private employees not directly mentioned and working during lockdown after June 14th",
+    home = 0.0,
+    # LAEC work = (rep(c(0, (1594000) / work_force, 0), c(3, 10, 3))),  #
+    work = (rep(c((1594000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.07 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_summer",
+    label = "Private employees not directly mentioned and working during the summer vacation",
+    home = 0.0,
+    work = (rep(c((1094000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.05 + private_residual_other_add,
+    risk = 1
+  )
+
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "remaining_private_companies_open",
+    label = "Private employees not directly mentioned",
+    home = 0.0,
+    work = (rep(c((1750000) / work_force, 0), c(13, 3))), #
+    school = 0,
+    other = 0.07 + private_residual_other_add,
+    risk = 1
+  )
+
+
+
+################################## #
+dk_activity_units <- dk_activity_units |>
+  add_activity(
+    activity = "home_225",
```

Comment:
Rename home_225 -> home_22_5
Repeat for other activities for consistency

Reply:
Renaming home_225 -> home_22_5pp

(In the rest of the activity unites, we often use "pp" to indicate percentage changes)